webapp/channels/src/plugins/channel_header_plug/channel_header_plug.test.tsx

```diff
@@ -61,11 +61,11 @@ describe('plugins/ChannelHeaderPlug', () => {
                 shouldShowAppBar={false}
             />,
         );
-        expect(wrapper).toMatchSnapshot();
+        expect(asFragment()).toMatchSnapshot();
     });
 
     test('should match snapshot with six extended components', () => {
```

Comment:
Out of scope, but the test case says 6 and we are passing 15.

Reply:
Looking back to when this was added, the intent of this test is to confirm that only a certain number of components would be rendered at once before the rest are pushed into a menu. I'll see how hard it is to make a non-snapshot test to reflect that

Reply:
Actually, I'm going to split this into a separate PR since I want to make some other changes here. It's Friday, so I feel more free to make random improvements today :P

Reply:
Also, turns out it was actually bumped to 15 🙄

Reply:
Addressed in https://github.com/mattermost/mattermost/pull/26784